module/Frontpage/src/Frontpage/Form/PollComment.php

```diff
@@ -55,8 +55,8 @@ public function getInputFilterSpecification()
                     [
                         'name' => 'string_length',
                         'options' => [
-                            'min' => 3,
-                            'max' => 128
+                            'min' => 2,
+                            'max' => 32
                         ]
```

Comment:
32 characters seems very short for a poll comment?

Reply:
This is the author name. I don't think we need long fields for that